target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -60,6 +60,7 @@ ramips_setup_interfaces()
 	planex,mzk-ex300np|\
 	planex,mzk-ex750np|\
 	ravpower,wd03|\
+	hootoo,tm05|\
```

Comment:
sorting.

Reply:
Ummm ... not sure what you mean / want. Can you clarify? Thanks!

Reply:
The device names should be sorted alphabetically.

Reply:
Got it - done!